packages/app/src/app/pages/common/Modals/ForkFrozenSandboxModal/index.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { observer } from 'mobx-react-lite';
+
+import Row from '@codesandbox/common/lib/components/flex/Row';
+import { Button } from '@codesandbox/common/lib/components/Button';
+import { useStore, useSignals } from 'app/store';
+import { Container } from '../LiveSessionEnded/elements';
+import { Heading, Explanation } from '../elements';
+import { Close } from './elements';
+
+const ForkFrozenSandboxModal = () => {
+  const {
+    editor: {
+      currentSandbox: { customTemplate },
+    },
+  } = useStore();
+  const { modalClosed, editor } = useSignals();
+
+  return (
+    <Container>
+      <Close onClick={() => modalClosed()} />
```

Comment:
@SaraVieira This can just be

```suggestion
      <Close onClick={modalClosed} />
```